tensorboard/plugins/hparams/tf_hparams_query_pane/tf-hparams-query-pane.html

```diff
@@ -741,6 +799,7 @@
           // If query configuration is not valid don't send a request.
           return;
         }
+        this.set('_sessionGroupsRequest', request);
```

Comment:
OK—it’s a bit awkward to put this here, but I agree that it’s about the
best that can be done given the current code organization, and it
wouldn’t be fair to ask you to fix the existing design as part of this
change, so this is fine with me.

Reply:
Yeah, I didn't love the solution, but it was the best I could think of.